src/frame.js

```diff
@@ -0,0 +1,73 @@
+function Frame(){
+  this.rolls = [0, 0];
+  this.MAX_ROLL = 10;
+  this.score = 0;
+}
+
+Frame.prototype.getRoll = function(number) {
+  return this.rolls[number - 1];
+};
+
+Frame.prototype.update = function(firstroll, secondroll) {
+  this.rolls[0] = firstroll;
+  this.rolls[1] = secondroll;
+};
+
+Frame.prototype.getAllRolls = function () {
+  return this.rolls;
+};
+
+Frame.prototype._rollSum = function () {
+  if (this.rolls[2] == undefined) {
+    return this.rolls[0] + this.rolls[1];
+
+  } else {
+      return this.rolls.reduce(function (total, roll) {
```

Comment:
Nice use of reduce.
You can also define a helper function called sum
```js
function sum(array){
    return array.reduce((a, b) => a + b)
}
```
and then call `sum(this.rolls)`